### PR TITLE
Add proxy support in signature updater

### PR DESCRIPTION
**Pull request by cccs-rs**
Changes to the UI for [Add proxy support to signature updater](https://cccs.atlassian.net/browse/AL-939)

Changes to odm are in this PR: https://github.com/CybercentreCanada/assemblyline-base/pull/48

### CODE REVIEW

File: assemblyline_ui/static/ng-template/signature_source.html [hunk 1]
```diff
@@ -69,6 +69,13 @@
 	<div id="conf_temp_error" class="col-md-12 text-danger">{{conf_temp_error}}</div>
 </section>
 
+<section class="row">
+	<div class="col-md-12" class="form-group" style="margin-top: 10px;">
+		<label class="control-label">Proxy</label>
+        <input class="form-control input-sm" ng-model="current_source.proxy" placeholder="scheme://user:pass@proxy:port"></input>
```
Comment (cccs-kevin) [inline assemblyline_ui/static/ng-template/signature_source.html:75]: From what I understand about HTML syntax, I think this tag would look better if it was inline with the <label> tag

**Review by cccs-kevin — APPROVED** [review]
🍨

**Comment by cccs-rs** [comment]
This PR will be closed; this addition to the UI will be part of new UI that @cccs-sgaron is working on.

**Comment by cccs-sgaron** [comment]
We can still have it in for now since the new UI is not ready. But this and the dependency patch will be the last ones of the old UI.

**Review by cccs-sgaron — APPROVED** [review]

**Pull request merged**